python_tests/test_python_frontend.py

```diff
@@ -50,10 +64,12 @@ def is_pre_ampere():
 
 
 def setUpModule():
-    from nvfuser import enable_automatic_serialization
+    if not debug_serde:
+        from nvfuser import enable_automatic_serialization
+
+        # Turn on default serialization upon program exit
+        enable_automatic_serialization()
```

Comment:
Instead of creating a hook to do that upon program exit, how about simply serialize in tearDownModule for less indirection?

Reply:
It is to test the automatic serialization/deserialization API. 
https://github.com/NVIDIA/Fuser/blob/main/csrc/python_frontend/fusion_cache.h#L212-L220